<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraThreadViewerGUI</class>
 <widget class="QMainWindow" name="CameraThreadViewerGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CameraThreadViewerGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="ImageViewerWidget" name="imageViewer"/>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="0">
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Lights</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QPushButton" name="pbLights">
              <property name="text">
               <string>Calibrate</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pbLightsTest">
              <property name="text">
               <string>Test</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="startButton">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="stopButton">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="title">
              <string>Image type</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="4" column="0">
               <widget class="QRadioButton" name="bRadioButton">
                <property name="text">
                 <string>B</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QRadioButton" name="rawRadioButton">
                <property name="text">
                 <string>Raw</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="Rgblabel">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="10" column="0">
               <widget class="QLabel" name="lblDark">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QRadioButton" name="rgbRadioButton">
                <property name="text">
                 <string>RGB</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QRadioButton" name="rRadioButton">
                <property name="text">
                 <string>R</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="FPSlabel">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QRadioButton" name="gRadioButton">
                <property name="text">
                 <string>G</string>
                </property>
               </widget>
              </item>
              <item row="9" column="0">
               <widget class="QLabel" name="lblImfSize">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="11" column="0">
               <widget class="QCheckBox" name="cbTrigger">
                <property name="text">
                 <string>Trigger</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pauseButton">
             <property name="text">
              <string>Pause</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Saving</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Save folder</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QLineEdit" name="saveFolderLineEdit"/>
                  </item>
                  <item>
                   <widget class="QToolButton" name="btnSaveFolder">
                    <property name="text">
                     <string>...</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="ClickableLabel" name="lblSaveFolder">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="0">
               <widget class="QToolButton" name="btnStartSave">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonIconOnly</enum>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QToolButton" name="btnEndSave">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QToolButton" name="btnSnap">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="lblCounter">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Focusing</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="0">
             <widget class="QPushButton" name="pbFocus">
              <property name="text">
               <string>Focus</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="btnFocusTest">
              <property name="text">
               <string>Focus test</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QSpinBox" name="sbFirgelliVal">
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>4000</number>
              </property>
              <property name="maximum">
               <number>32767</number>
              </property>
              <property name="singleStep">
               <number>250</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="zaxisMsgBtn">
              <property name="text">
               <string>Focus Msg</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblError">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageViewerWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">imageviewerwidget.h</header>
  </customwidget>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>camerathreadviewergui.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
